docs/reference/extra.rst

```diff
@@ -158,6 +157,8 @@ First of all, you have to define your ``IvoryCKEditorBundle`` (already embedded
 
 .. code-block:: yaml
 
+    # config/packages/ivory_ck_editor.yaml
+
     ivory_ck_editor:
```

Comment:
should this be fos_ck_editor now?

Reply:
not sure, @core23 ?

Reply:
Ping @core23

Reply:
done, copied this part from formatter bundle docs